substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/image/NativeImageDebugInfoProvider.java

```diff
@@ -958,11 +976,33 @@ public int getFrameSize() {
 
         @Override
         public boolean isDeoptTarget() {
-            return methodName().endsWith(HostedMethod.METHOD_NAME_DEOPT_SUFFIX);
+            return name().endsWith(HostedMethod.METHOD_NAME_DEOPT_SUFFIX);
+        }
+
+        @Override
+        public List<String> paramTypes() {
+            LinkedList<String> paramTypes = new LinkedList<>();
```

Comment:
What's the point of using linked list (here and elsewhere). Using linked list is almost always a bad idea.

Reply:
There's no excuse for it... Thanks for catching it!